### ISSUE TITLE
Hightlite the simulator bar on the main page

### ISSUE THREAD

**Issue by Malavi1** [CircuitVerse/CircuitVerse#3534]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

As the main aim of the circuit verse is simulation,Even though we are highlighting the simulator navigation bar, with hovering and border effect it is some how less **notice** in user perspective.

![Screenshot (211)1](https://user-images.githubusercontent.com/112646623/215278523-477a336e-2140-4513-bdfe-356c496977a8.png)

**Describe alternatives you've considered**
I just make a try to seek the attention of the user by changing the background color into the circuit verse default color which is #42b983 like below 

![Screenshot (212)1](https://user-images.githubusercontent.com/112646623/215279127-8b92e939-6356-434e-8d52-795b4cfa7e15.png)

**Are you working on this? (Yes/No)**
Yes
shall I proceed on this ?

**Comment by adi-develops** [comment]
Hi @Malavi1 ! I would like to contribute to this issue. Can you please assign this issue to me.

**Comment by Malavi1** [comment]
@adi-develops I am also looking for approval & assign, good to rely on other than this

**Comment by vedant-jain03** [comment]
@Malavi1 thanks for the suggestion but we have added the simulator button UI intentionally, if you see we have not used the similar UI for any other button, we have made it standardise with the navbar color(primary and white) and these are to make it unique, simple. The navbar as we know is `fixed` as well.

I don't think we need to change UI for the button now, although the work to make the navbar UI attractive is under progress, we will take your suggestion into consideration!

Thankyou!
